### DISCUSSION TITLE
Is it possible to generate RSS/Atom/JSON feed for docs?

### DISCUSSION THREAD

**Discussion by jkboxomine** [facebook/docusaurus#8916] [Q&A]
Hi,

It appears that generating RSS/Atom/JSON feed for Blog is supported with configuration.
https://docusaurus.io/docs/blog#feed

Is it possible to add such configuration or implement the feed generation feature for Docs? (even with any workaround)

**Comment by slorber** [comment]
We don't have this feature and I'm not sure how it would work. How would you define the feed order for docs?

**Comment by stefanpejcic** [comment]
Same order as defined in sidebar.js
If multiple sidebars, each should be a separate .xml

**Reply by slorber** [reply]
feeds are usually ordered chronologically IMHO.
Don't you think most users would usually expect a newly created doc should rather be the first doc of the feed?


---

If you want this feature to be implemented, please give great details on your business usecase for it. It's important we understand it to design this feature properly.
